HW1/task3.py

```diff
@@ -0,0 +1,14 @@
+# Вам требуется написать программу, которая проверяет счастливость билета. Cумма первых трех цифр равна сумме последних трех.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+
+str = input("Введите шестизначное число:")
+
+if len(str) != 6 :
+    print("Вы ввели не шестизначное число!")
+else :
+    if (int(str[0]) + int(str[1]) + int(str[2])) == (int(str[3]) +int(str[4]) + int(str[5])) :
+        print("Да, Это счастливый билет!")
+    else :
+        print("Нет, Это несчастливый билет!")
```

Comment:
Если после `else` идет `if`, то можно сделать сокращение до `elif`. И внешние скобки применять не обязательно.
```suggestion
elif int(str[0]) + int(str[1]) + int(str[2]) == int(str[3]) +int(str[4]) + int(str[5]):
    print("Да, Это счастливый билет!")
else :
    print("Нет, Это несчастливый билет!")
```